cmd/admin_build_instance.go

```diff
@@ -54,12 +52,12 @@ func adminBuildInstanceCommand(conf *config.ClientConfig) *cli.Command {
 	cmd.Flags().StringVar(&runName, "name", "", "Name of running instance, e.g., bq2bq/transporter/predator")
 	cmd.MarkFlagRequired("name")
 
-	cmd.Flags().StringVarP(&projectName, "project", "p", projectName, "Name of the optimus project") // TODO: fix overriding conf via args
-	cmd.Flags().StringVar(&optimusHost, "host", optimusHost, "Optimus service endpoint url")         // TODO: fix overriding conf via args
+	cmd.Flags().StringP("project-name", "p", defaultProjectName, "Name of the optimus project")
```

Comment:
this change needs a corresponding change in the entrypoint.sh where ever it is referred when the optimus binary is upgraded in the plugins.

Reply:
noted, after we bump the optimus version, we can later update the flag name in `endpoint.sh` for each plugins